src/network/networkd-network.h

```diff
@@ -68,6 +68,14 @@ typedef enum LLDPMode {
         _LLDP_MODE_INVALID = -1,
 } LLDPMode;
 
+typedef struct DUID {
+        /* Value of Type in [DHCP] section */
+        DUIDType type;
+
+        uint8_t duid_len;
+        uint8_t duid[MAX_DUID_LEN];
+} DUID;
```

Comment:
I thought about the name too. It was either `duid` or  `data` or `raw_data` to match the config option name. I'll rename it.